openbb_terminal/miscellaneous/sources/openbb_default.json

```diff
@@ -896,6 +896,9 @@
       ],
       "tx": [
         "Ethplorer"
+      ],
+      "solana": [
```

Comment:
Here you have assigned a function named "solana", but you have called it "topledger"

![Screenshot 2023-05-19 at 1 53 27 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/8d8562e4-ffec-40f3-affd-986bd8cd2769)

Reply:
the key should be the name of the command. it's here that we tell what sources each commands have. so if you replace `solana` with `topledger` it will display in the onchain menu topledger  ....   [TopLedger]

hence my recommendation to call the command something else like tlquery (since we already have a query)

Reply:
thanks, have updated the command name to `topledger`